torch/_dynamo/variables/torch.py

```diff
@@ -1646,162 +1641,6 @@ def handle_ntuple(value):
         else:
             return handle_ntuple(args[0])
 
-    @classmethod
-    def call_nn_parameter(cls, tx, data=None, requires_grad=True):
-        """A call to torch.nn.Parameter() gets lifted to before the graph"""
-        if tx.export:
-            unimplemented_v2(
-                gb_type="Attempted to use `torch.nn.Parameter()` with export",
-                context="",
-                explanation="Dynamo does not support this.",
-                hints=[
-                    "Do not use `torch.nn.Parameter()` with export.",
-                    *graph_break_hints.SUPPORTABLE,
-                ],
-            )
-
-        if isinstance(requires_grad, variables.VariableTracker):
-            try:
-                requires_grad = requires_grad.as_python_constant()
-            except NotImplementedError:
-                unimplemented_v2(
-                    gb_type="non-constant `requires_grad` argument to `torch.nn.Parameter`",
-                    context=f"requires_grad={requires_grad}",
-                    explanation="Dynamo does not support this.",
-                    hints=[
-                        "Change `requires_grad` to be a bool.",
-                        *graph_break_hints.USER_ERROR,
-                    ],
-                )
-
-        if not isinstance(data, variables.TensorVariable):
-            unimplemented_v2(
-                gb_type="`torch.nn.Parameter()` with unsupported data type",
-                context=f"data={data}",
-                explanation="Called `torch.nn.Parameter()` with non-Tensor argument.",
-                hints=[
-                    "Ensure the argument to `torch.nn.Parameter()` is a `torch.Tensor`.",
-                    *graph_break_hints.USER_ERROR,
-                ],
-            )
-
-        # this results in cleaner graphs, but only works for inputs
-        if data.source:
-            return cls._nn_param_via_prefix_insert(tx, data, requires_grad)
```

Comment:
I am not sure if it will reduce the blast radius. But can you try inserting the graph break after this line? This will ensure that sourceful params are still supported.